connect/src/com/telenor/connect/ui/ConnectWebViewLoginButton.java

```diff
@@ -23,16 +23,14 @@
 
 import androidx.annotation.NonNull;
 
-public class ConnectWebViewLoginButton extends ConnectButton implements AuthenticationButton {
-
-    public static final int NO_CUSTOM_LAYOUT = -1;
+class ConnectWebViewLoginButton extends ConnectButton implements AuthenticationButton {
 
     private ArrayList<String> acrValues;
     private Map<String, String> loginParameters;
     private ArrayList<String> loginScopeTokens;
     private int requestCode = 0xa987;
     private Claims claims;
-    private int customLoadingLayout = NO_CUSTOM_LAYOUT;
+    private int customLoadingLayout = -1;
```

Comment:
When removing the constant, can you remove this variable as well? In any case I'm a bit worried that you now have -1 two different places in the code.

Reply:
@vlarsen there is a big question if this part is needed at all. 

However, the reason for removing constant is to remove dependency that is now not accessible. Removing options for custom layout is a task of different scope that can be done later.